tests/unit/models/test_framework.py

```diff
@@ -0,0 +1,87 @@
+import pytest
+
+from consultation_analyser import factories
+from consultation_analyser.consultations import models
+
+
+def set_up_frameworks():
```

Comment:
A bit of a nit, but this is edging on "mystery guests" in the tests, where to read and understand the tests we just have to know that the different frameworks this method returns have particular relationships with each other. Giving them more explicit names, and maybe breaking them into fixtures, could help with this? that said, I'm not sure how whether pytest fixtures can depend on each other, so it might be that this is the pragmatic approach!

Reply:
Yes, will think about how to do this better. With some of the changes we discussed on Wed, have to change the factories anyway, so will be changing the tests.

Reply:
Have totally rewritten these tests with subsequent changes: https://github.com/i-dot-ai/consult/blob/feature/updated-models/tests/unit/models/test_framework.py - any thoughts on how to improve?